### ISSUE TITLE
[CLI] `db --migration-directory` option ignored

### ISSUE THREAD

**Issue by cedx** [stablekernel/aqueduct#398]
When using the `db generate`, `db migrate` or `db validate` commands of the Aqueduct CLI, I can't set the migration directory. It seems that this option is ignored:

```shell
/src/dart-project $ aqueduct db validate --migration-directory=lib/src/data

-- Aqueduct CLI Version: 2.5.0+1
-- Aqueduct project version: 2.5.0+1
*** No migration files found in /src/dart-project/migrations.
```

```shell
/src/dart-project $ aqueduct db generate --migration-directory=lib/src/data

-- Aqueduct CLI Version: 2.5.0+1
-- Aqueduct project version: 2.5.0+1
-- Replaying migration files...
-- The following ManagedObject<T> subclasses were found:
    [...]
-- Created new migration file (version 1).
    New file is located at /src/dart-project/migrations/00000001_Initial.migration.dart
```

I'm using the Dart VM version: 2.0.0-dev.32.0.

**Comment by joeconwaystk** [comment]
Looks like this flag used to be used, but it's behavior was removed without removing the option from the CLI. 

I'm leaning towards just removing the option - do you have a good use case to have another migration directory other than the default? Keeping in mind that migration files should be checked into version control.

**Comment by cedx** [comment]
A good use case? Not really: it's just that I usually have some directory layout, and that I prefer to have the migration folder into a `lib` subfolder (or whatever) instead of at the project root.

I can't do that currently... unless I use a symbolic link (i.e. `migrations` => `lib/migrations`).
Conversely, I can do that without any issue with the YAML configuration files (i.e. store them into an `etc` subfolder).

**Comment by joeconwaystk** [comment]
👍 We'll get that fixed in 3.0.

**Comment by cedx** [comment]
Thanks a lot :)

**Comment by joeconwaystk** [comment]
Fixed in 3.0 branch
